README.md

```diff
@@ -26,4 +26,4 @@ More Information
 Contributing
 ============
 
-For information on how to contribute code changes to MOOSE please [see the mooseframework.org wiki](http://mooseframework.org/wiki/Contributing/).
+For information on how to contribute code changes to MOOSE please [see this article](http://mooseframework.org/framework_development/contributing.html).
```

Comment:
I'd prefer a relative path since you are fixing it anyway. If it's a full path, it probably should be "https".